src/styles/blocks/catalog.scss

```diff
@@ -11,15 +13,22 @@
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
+    cursor: pointer;
 
     width: 200px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all $transition-time;
```

Comment:
don't use all in your transitions, better specify all properties to make them refactorable in future